### PR TITLE
TNO-1133: Front pages

### PR DESCRIPTION
**Pull request by jdtoombs**
This PR Includes the feature that will display todays 5 front pages in the box located on the lower right hand side of the landing screen.

### CODE REVIEW

File: libs/net/dal/Services/ContentService.cs [hunk 1]
```diff
@@ -175,6 +175,23 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
         return new Paged<Content>(items, filter.Page, filter.Quantity, total);
     }
 
+    /// <summary>
+    /// Find today's front pages
+    /// </summary>
+    /// <returns>The 5 front pages for today's date.</returns>
+    public IPaged<Content> FindFrontPages(bool asNoTracking = true)
```
Comment (jdtoombs) [inline libs/net/dal/Services/ContentService.cs:182]: Is this overkill?

Comment (Fosol) [reply libs/net/dal/Services/ContentService.cs:182]: Use Elasticsearch not the database.

**Review by Fosol — CHANGES_REQUESTED** [review]

**Review by Fosol — CHANGES_REQUESTED** [review]
I've also published `tno-core:0.0.259`.  You'll need to publish a new version.

**Comment by jdtoombs** [comment]
@Fosol - this PR has been updated

File: libs/net/dal/Services/ContentService.cs [hunk 2]
```diff
@@ -174,7 +174,34 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
...
+    /// <summary>
+    /// Find most recent front pages (5 of them).
+    /// </summary>
+    /// <returns> Up to 5 front pages.</returns>
+    public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindFrontPages(string index)
+    {
+        var productQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
+        var today = DateTime.Today.ToUniversalTime();
+        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 10}}"));
+         var response = await _client.SearchAsync<API.Areas.Services.Models.Content.ContentModel>(s =>
```
Comment (Fosol) [inline libs/net/dal/Services/ContentService.cs:186]: Not a big deal, but you chose to use the NEST library instead of sending a raw JSON request.

**Review by Fosol — APPROVED** [review]

**Pull request merged**